library/ecs_service.py

```diff
@@ -268,12 +272,26 @@ def create_service(self, service_name, cluster_name, task_definition,
         return self.jsonize(response['service'])
 
     def update_service(self, service_name, cluster_name, task_definition,
-                       load_balancers, desired_count, client_token, role):
-        response = self.ecs.update_service(
+                       load_balancers,
+                       desired_count, deployment_maximum_percent, deployment_minimum_healthy_percent,
+                       client_token, role):
+        # Boto doesn't like it if you pass defaults for parameters you aren't actually using
+        params = dict(
             cluster=cluster_name,
             service=service_name,
             taskDefinition=task_definition,
             desiredCount=desired_count)
+
+        if deployment_maximum_percent is not None or deployment_minimum_healthy_percent is not None:
+            params['deploymentConfiguration'] = {}
```

Comment:
Does this logic work?
If you don't set `deployment_maximum_percent` or `deployment_minimum_healthy_percent` should the behaviour be to leave the existing ECS settings as is or reset them to empty values?